cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -0,0 +1,116 @@
+        sleep 10
+    done
+}
+
+function deploy_sriov_operator {
+  operator_path=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/sriov-network-operator-${OPERATOR_GIT_HASH}
+  if [ ! -d $operator_path ]; then
+    curl -L https://github.com/openshift/sriov-network-operator/archive/${OPERATOR_GIT_HASH}/sriov-network-operator.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
+  fi
+
+  pushd $operator_path
+    export RELEASE_VERSION=4.4
+    export SRIOV_NETWORK_OPERATOR_IMAGE=quay.io/openshift/origin-sriov-network-operator:${RELEASE_VERSION}
+    export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=quay.io/openshift/origin-sriov-network-config-daemon:${RELEASE_VERSION}
+    export SRIOV_NETWORK_WEBHOOK_IMAGE=quay.io/openshift/origin-sriov-network-webhook:${RELEASE_VERSION}
+    export NETWORK_RESOURCES_INJECTOR_IMAGE=quay.io/openshift/origin-sriov-dp-admission-controller:${RELEASE_VERSION}
+    export SRIOV_CNI_IMAGE=quay.io/openshift/origin-sriov-cni:${RELEASE_VERSION}
+    export SRIOV_DEVICE_PLUGIN_IMAGE=quay.io/openshift/origin-sriov-network-device-plugin:${RELEASE_VERSION}
+    export OPERATOR_EXEC=${KUBECTL}
+    export SHELL=/bin/bash  # on prow nodes the default shell is dash and some commands are not working
+    make deploy-setup-k8s
+  popd
+
+  pushd "${CSRCREATORPATH}" 
+    go run . -namespace sriov-network-operator -secret operator-webhook-service -hook operator-webhook
+    go run . -namespace sriov-network-operator -secret network-resources-injector-secret -hook network-resources-injector
+  popd
+}
+
+if [[ -z "$(_kubectl get nodes | grep $FIRST_WORKER_NODE)" ]]; then
+  SRIOV_NODE=$MASTER_NODE
+else
+  SRIOV_NODE=$FIRST_WORKER_NODE
+fi
+
+#move the pf to the node
+mkdir -p /var/run/netns/
+export pid="$(docker inspect -f '{{.State.Pid}}' $SRIOV_NODE)"
+ln -sf /proc/$pid/ns/net "/var/run/netns/$SRIOV_NODE"
+
+sriov_pfs=( /sys/class/net/*/device/sriov_numvfs )
+
+
+for ifs in "${sriov_pfs[@]}"; do
+  ifs_name="${ifs%%/device/*}"
+  ifs_name="${ifs_name##*/}"
+
+  if [ $(echo "${PF_BLACKLIST[@]}" | grep -q "${ifs_name}") ]; then
+    continue
+  fi
+
+  # We set the variable below only in the first iteration as we need only one PF
+  # to inject into the Network Configuration manifest. We need to move all pfs to
+  # the node's namespace and for that reason we do not interrupt the loop.
+  if [ -z "$NODE_PF" ]; then
+    # These values are used to populate the network definition policy yaml.
+    # We just use the first suitable pf
+    # We need the num of vfs because if we don't set this value equals to the total, in case of mellanox
+    # the sriov operator will trigger a node reboot to update the firmware
+    export NODE_PF="$ifs_name"
+    export NODE_PF_NUM_VFS=$(cat /sys/class/net/"$NODE_PF"/device/sriov_totalvfs)
+  fi
+  ip link set "$ifs_name" netns "$SRIOV_NODE"
+done
+
+
+# deploy multus
+_kubectl apply -f https://raw.githubusercontent.com/intel/multus-cni/master/images/multus-daemonset.yml
```

Comment:
You should download the file and replace it with the old one instead of download it,
so in case the file you download changes nothing will break.